lua/tests/gmod/classes/file/__gc.lua

```diff
@@ -0,0 +1,34 @@
+file.CreateDir("file_tests")
+function GetTestFile( name, write )
+	local f = file.Open( "file_tests/" .. name .. ".txt", write and "wb" or "rb", "DATA" )
+	if !f then error( "Missing test file! (" .. name .. ")" ) end
+
+	return f
+end
```

Comment:
You could move this over to `lua/gmod_tests/sh_init.lua`, or maybe `lua/gmod_tests/utils/file.lua`

I made that new directory to store helper functions like this one!

Also, some LuaLS annotations would be TOPS:
```suggestion
local testPath = "gmod_tests_file_tests"
file.CreateDir( testPath )

--- Returns a new File object for use in the gmod_tests test suite
--- @param name string The name of the file in the test directory
--- @param write boolean Whether or not to open the File in "write" mode
--- @returns File
function GetTestFile( name, write )
        local path = testPath .. "/" .. name .. ".txt"
        local mode = write and "wb" or "rb"
        
	local f = file.Open( path, mode, "DATA" )
	if f then error( "Missing test file! (" .. path .. ")" ) end

	return f
end
```